src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,71 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
+function isHello(val1) {
   // TODO: write code in this function body to pass the tests
-
+  if (val1 === "Hello") {
+    return true;
+  }
+  if (val1 !== "Hello") {
+    return false;
+  }
 }
-
 // This function should return true if the passed string is not equal to "Hello"
-function isNotHello (val1) {
-
+function isNotHello(val1) {
   // TODO: write code in this function body to pass the tests
-
+  if (val1 !== "Hello") {
```

Comment:
Same as above, you can simplify this by returning `val1 !== 'Hello'`. Then you wouldn't need the second if statement.